doc/architecture/decisions/0006-splitting-agent-into-front-house-back-house-or-not.md

```diff
@@ -0,0 +1,38 @@
+# 6. Splitting agent into front-house / back-house - OR NOT
```

Comment:
Will we change the filename once we change the status to Accepted or Rejected?

Reply:
I would not. Why should we?